.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
-      - "6.0.0"
-before_script:
-      - npm install
```

Comment:
travis already runs npm install (and npm run build) when you make the language node_js, so we don't need to.